### PR TITLE
Add missing documentation on SBO installation

### LINKED ISSUE #5757: Missing documentation on service binding in 3.0.0 docs
/kind documentation

## What mistake did you find / what is missing in the documentation?

Here:  https://odo.dev/docs/3.0.0/command-reference/add-binding

There is this part:

`A cluster with the Service Binding operator installed, along with the operator whose service you need to bind to`

We should be pointing to a link on how to install the service binding operator, which should be here: https://odo.dev/docs/3.0.0/overview/cluster-setup/kubernetes

We need to re-add the instructions on how to add service binding operator from the 2.5.0 page: https://odo.dev/docs/getting-started/cluster-setup/kubernetes

## What is the relevance of it?

Missing docs

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**
/kind documentation

**What does this PR do / why we need it:**
This PR adds missing documentation on the service binding operator.
**Which issue(s) this PR fixes:**

Fixes #5757 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [x] Documentation 

**How to test changes / Special notes to the reviewer:**
https://deploy-preview-5855--odo-docusaurus-preview.netlify.app/docs/3.0.0/command-reference/add-binding#pre-requisites

https://deploy-preview-5855--odo-docusaurus-preview.netlify.app/docs/3.0.0/overview/cluster-setup/kubernetes#installing-the-service-binding-operator

https://deploy-preview-5855--odo-docusaurus-preview.netlify.app/docs/3.0.0/overview/cluster-setup/openshift/#installing-the-service-binding-operator

### CODE REVIEW

File: docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md [hunk 1]
```diff
@@ -16,7 +16,13 @@ This guide is helpful in setting up a development environment intended to be use
 * You have a Kubernetes cluster set up (such as [minikube](https://minikube.sigs.k8s.io/docs/start/))
 * You have admin privileges to the cluster
 
-**Important notes:** `odo` will use the __default__  storage provisioning on your cluster. If it have not been set correctly, see our [troubleshooting guide](/docs/getting-started/cluster-setup/kubernetes#troubleshooting) for more details.
+**Important notes:** `odo` will use the __default__  storage provisioning on your cluster. If it has not been set correctly, see our [troubleshooting guide](/docs/getting-started/cluster-setup/kubernetes#troubleshooting) for more details.
+
+## Installing the Service Binding Operator
```
Comment (cdrage) [inline docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md:21]: We should add in brackets (OPTIONAL) for this section.

Comment (cdrage) [inline docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md:21]:
```suggestion
## (OPTIONAL) Installing the Service Binding Operator
```

File: docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/openshift.md [hunk 2]
```diff
@@ -12,3 +12,8 @@ This guide is helpful in setting up a development environment intended to be use
 * You have a OpenShift cluster set up (such as [crc](https://crc.dev/crc/#installing-codeready-containers_gsg))
 * You have admin privileges to the cluster
 
+## Installing the Service Binding Operator
```
Comment (cdrage) [inline docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/openshift.md:15]:
```suggestion
## (OPTIONAL) Installing the Service Binding Operator
```

**Review by cdrage — CHANGES_REQUESTED** [review]

File: docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md (hunk 1 above)
Comment (valaparthvi) [reply docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md:21]: It's not optional if user want to use binding.

**Comment by feloy** [comment]
/lgtm

/override validator/Validate
/override OpenShift Integration tests/OpenShift Integration tests
/override Kubernetes Integration Tests/Kubernetes Integration Tests

Doc only

**Pull request merged**